Assets/Scenes/LevelPrototypeScene.unity

```diff
@@ -20,11 +20,11 @@ RenderSettings:
   m_FogDensity: 0.01
   m_LinearFogStart: 0
   m_LinearFogEnd: 300
-  m_AmbientSkyColor: {r: 0.212, g: 0.227, b: 0.259, a: 1}
+  m_AmbientSkyColor: {r: 0.5874866, g: 0.64951944, b: 0.7735849, a: 1}
```

Comment:
what happened here

Reply:
I had to hardcode the lighting color in order to fix the dark lighting issue.

Reply:
Again dont expect it to be an issue later on